adding-more-native-to-your-react-native-app.md

```diff
@@ -0,0 +1,186 @@
+# React Native Link
+
+### Who am I?
+
+- Eloy Durán
+- @alloy
+- Author of CocoaPods
+- Engineer at Artsy, where we’ve been using React Native for 3.5 years in our brownfield app
```

Comment:
@bartolkaruza We should also include your pluggables somewhere. Perhaps on the final slide that we can leave up during QA? What details do you want to include?